Update the change log for v0.6.0 (#535)
aws/eks-anywhere: docs/content/en/docs/reference/changelog.md

---
linkTitle: "Changelog"
weight: 55

> ---

Reviewer: I think we need the header so this shows up in the TOC
![Screen Shot 2021-11-01 at 14 55 21](https://user-images.githubusercontent.com/104113/139741014-0345bb92-fc75-4d04-8e98-5101141fe1df.png)

Not sure why it was removed

Author: was removed by mistake! added it back.

### Added

- aws-iam-authenticator support
- Allow the retention of node taints during an upgrade

## v0.6.0 - 2021-10-29

> ### Added

Reviewer 2: Do we have tasks or concepts documented for any of these features? It would be great to be able to link readers back to our docs on getting started with each new feature.

Reviewer 3: > Do we have tasks or concepts documented for any of these features? It would be great to be able to link readers back to our docs on getting started with each new feature.

I'll be writing up content related to the "management cluster". I know there are also docs on the "bring your own registry" feature and possibly for the upgrade improvements. I agree we should add links to those docs here once they are locked down.

Author: yes we do, I was deciding on whether we should add the issue or the prs' links. I will add the issue and docs links.

> - Support to create and manage a management cluster

Reviewer 4: I believe this could be 

```
Support to create and manage workload clusters
```

@jaxesn @vipin-mohan ?

> linkTitle: "Changelog"

Reviewer 3: Sorry, I think I wasn't clear before. You can remove lines 2, 3, and 4, I think.

weight: 55
---

# What's New?

## [Unreleased]

### Added

- aws-iam-authenticator support
- Allow the retention of node taints during an upgrade

## v0.6.0 - 2021-10-29

### Added

- Support to create and manage workload clusters [#94](https://github.com/aws/eks-anywhere/issues/94)
- Support for upgrading eks-anywhere components [#93](https://github.com/aws/eks-anywhere/issues/93), [cluster upgrades](https://github.com/aws/eks-anywhere/blob/main/docs/content/en/docs/tasks/cluster/cluster-upgrades.md)
- k8s CIS compliance [#193](https://github.com/aws/eks-anywhere/pull/192/)
- Support bundle improvements [#92](https://github.com/aws/eks-anywhere/issues/92)
- Ability to upgrade control plane nodes before worker nodes [#100](https://github.com/aws/eks-anywhere/issues/100)

> - Ability to "bring your own registry" [#98](https://github.com/aws/eks-anywhere/issues/98)

Reviewer 5: I would say "Ability to use your own container registry" instead
```suggestion
- Ability to use your own container registry [#98](https://github.com/aws/eks-anywhere/issues/98)
```

> - Make namespace configurable for anywhere resources [#177](hsuttps://github.com/aws/eks-anywhere/pull/177/files)

Reviewer 5: fix broken link
```suggestion
- Make namespace configurable for anywhere resources [#177](https://github.com/aws/eks-anywhere/pull/177/files)
```

> - Add proxy configuration to etcd nodes for bottlerocket [#195](https://github.com/aws/eks-anywhere/issues/195)

Reviewer 5: I would move this to the "Fixed" section since it's not a new feature but a fix for a pre-existing feature

weight: 55

> ---

Reviewer 3: I'd like to add a link to this page to the menu bar. I think changing the header to look like this would work:
`title: "What's New?"
linkTitle: "What's New?"
weight: 55
menu:
  main:
    weight: 10
---
`

Reviewer 3: So this would replace the two lines of three-dashes and everything in between.

# What's New?

## [Unreleased]

### Added

- aws-iam-authenticator support
- Allow the retention of node taints during an upgrade

## v0.6.0 - 2021-10-29

### Added

- Support to create and manage workload clusters [#94](https://github.com/aws/eks-anywhere/issues/94)

> - Support for upgrading eks-anywhere components [#93](https://github.com/aws/eks-anywhere/issues/93), [cluster upgrades](https://github.com/aws/eks-anywhere/blob/main/docs/content/en/docs/tasks/cluster/cluster-upgrades.md)

Reviewer 3: ```suggestion
- Support for upgrading eks-anywhere components [#93](https://github.com/aws/eks-anywhere/issues/93), [cluster upgrades] [Create cluster workflow]({{< relref "/docs/tasks/cluster/cluster-upgrades" >}})
```
This form of links gets checked by hugo.
